app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
these are additional features we can add via devise. Let's add confirmable, trackable. 

You need to comment out these migration columns from the devise generated migration file. To run a migration again, you can rollback. Have a look at `rake db:rollback` which would undo the last migration. You can do that to rollback and then go forward again.